crates/core/tedge_agent/src/tedge_operation_converter/actor.rs

```diff
@@ -43,7 +43,7 @@ pub struct TedgeOperationConverterActor {
     pub(crate) workflows: WorkflowSupervisor,
     pub(crate) state_repository: AgentStateRepository<CommandBoard>,
     pub(crate) log_dir: Utf8PathBuf,
-    pub(crate) input_receiver: LoggingReceiver<AgentInput>,
+    pub(crate) input_receiver: UnboundedLoggingReceiver<AgentInput>,
```

Comment:
An alternative to fix the issue could be to use two receivers.

- One receiver to receive progress feedback from the software actor
- Another to collect requests from MQTT

Using a biased select, one can then give the priority to feedback messages. 

 with a biased `select!`